dom.bs

```diff
@@ -4919,17 +4919,40 @@ invoked, must run these steps:
    <thead>
     <tr><th>String<th>Interface<td>Notes
    <tbody>
-    <tr><td>"<code>customevent</code>"<td>{{CustomEvent}}<td rowspan=4>
-    <tr><td>"<code>event</code><td rowspan=3>{{Event}}
+    <tr><td>"<code>animationevent</code>"<td>{{AnimationEvent}}<td>[[!CSS3-ANIMATIONS]]
+    <tr><td>"<code>beforeunloadevent</code>"<td>{{BeforeUnloadEvent}}<td rowspan=2>[[!HTML]]
+    <tr><td>"<code>closeevent</code>"<td>{{CloseEvent}}
+    <tr><td>"<code>compositionevent</code>"<td>{{CompositionEvent}}<td>[[!UIEVENTS]]
+    <tr><td>"<code>customevent</code>"<td>{{CustomEvent}}<td>
+    <tr><td>"<code>devicemotionevent</code>"<td>{{DeviceMotionEvent}}<td rowspan=2>[[!DEVICE-ORIENTATION]]
+    <tr><td>"<code>deviceorientationevent</code>"<td>{{DeviceOrientationEvent}}
+    <tr><td>"<code>dragevent</code>"<td>{{DragEvent}}<td rowspan=2>[[!HTML]]
+    <tr><td>"<code>errorevent</code>"<td>{{ErrorEvent}}
+    <tr><td>"<code>event</code>"<td rowspan=2>{{Event}}<td rowspan=2>
     <tr><td>"<code>events</code>"
-    <tr><td>"<code>htmlevents</code>"
+    <tr><td>"<code>focusevent</code>"<td>{{FocusEvent}}<td>[[!UIEVENTS]]
+    <tr><td>"<code>hashchangeevent</code>"<td>{{HashChangeEvent}}<td>[[!HTML]]
+    <tr><td>"<code>htmlevents</code>"<td>{{Event}}<td>
```

Comment:
I considered that, but decided it would be confusing if the reader were trying to check if an event is on the list, and that that outweighs the benefit.